guidelines/sc/20/parsing.html

```diff
@@ -1,8 +1,31 @@
 <section class="sc">
    
-   <h4>Parsing</h4>
+   <h4>Parsing (Obsolete)</h4>
+
+   <p class="conformance-level">A</p>
+
+   <p>In content implemented using markup languages, elements have complete start and end
+      tags, elements are nested according to their specifications, elements do not contain
+      duplicate attributes, and any IDs are unique, except where the specifications allow
+      these features.
+   </p>
+
+   <div class="note">
+      <p>
+         Modern web technologies have standardized how user agents parse incorrect markup. 
+         Any invalid markup is therefore allowed under 4.1.1 Parsing for technologies such as
+         HTML 5 and CSS 3. This success criterion is always satisfied for these technologies. 
+         This success criterion is therefore obsolete.
+      </p><p>
+         Issues such as incorrect states or names due to a duplicate ID, or missing roles 
+         due to nesting interactive elements are covered by different success criteria.
```

Comment:
not just due to nesting of interactive elements, but invalid nesting of elements in general can result in certain element's roles being dropped by the browser, or if not dropped, still not respected by AT due to their unexpected location in the a11y tree.